### ISSUE TITLE
[Bug] Slow performance with many cases

### ISSUE THREAD

**Issue by cugu** [TheHive-Project/TheHive#1721]
### Work Environment

| Question              | Answer
|---------------------------|--------------------
| CPU | 16 Cores
| RAM | 16 GB
| Package Type              | Docker
| Tag   | thehive4:4.0.2-1

### Problem Description

We imported about 5000 cases (out of 16000) cases into TheHive 4. The setup is using the Cassanda backend. Loading the main page takes quite long (~10 seconds).

### Steps to Reproduce
1. Import 5000 cases.
1. Open TheHive.

### Complementary information

Just retrieving the case count takes 4-7 seconds:

``` bash
time curl https://thehive.xxx.com/api/v1/query -s -H "Authorization: Bearer XXX" -H "Content-type: application/json" -d '{"query": [{"_name": "listCase"},{"_name": "count"}]}'
5190
real	0m5.174s
user	0m0.019s
sys	0m0.012s
```

This is similar to #1584 but related to cases this time.

**Comment by shortstack** [comment]
having the same issue. 8 cores, 32g. 4k cases, ~10+ seconds to load. alerts page is even slower.

**Comment by pemontto** [comment]
@nadouani would it be worth having a meta ticket to track all the TH4 performance issues?

https://github.com/TheHive-Project/TheHive/issues/1341
https://github.com/TheHive-Project/TheHive/issues/1465
https://github.com/TheHive-Project/TheHive/issues/1505
https://github.com/TheHive-Project/TheHive/issues/1506
https://github.com/TheHive-Project/TheHive/issues/1584
https://github.com/TheHive-Project/TheHive/issues/1721

**Comment by rriclet** [comment]
This issue has been fixed with #1731, included in release 4.1.0.
Feel free to re-open this issue if you still notice slow performance.
